### PR TITLE
fix: Broken scroll with scrollIndex

### PR DESCRIPTION
**Pull request by chiefcll**
## Description

When scrollIndex was set and scroll auto the row / column would never go back to the original start. Fixed that.

Cleaned up renaming of props, gap is gap, transition is transition - that occurs automatically with {...props}

Turned on inspector

refactored the withScrolling function to be cleaner and easier to read.

### CODE REVIEW

**Comment by chiefcll** [comment]
@anthony9187 - could you also look into why Column and Row don't scroll nicely - I assume it's missing a default transition value on x / y to animate

File: packages/solid/components/Row/Row.tsx [hunk 1]
```diff
@@ -31,9 +31,6 @@ const Row: Component<RowProps> = (props: RowProps) => {
     <View
       {...props}
       selected={props.selected || 0}
-      gap={props.itemSpacing}
-      transition={props.itemTransition}
-      scrollIndex={props.scrollIndex}
```
Comment (anthony9187) [inline packages/solid/components/Row/Row.tsx:36]: these need to be removed from the interface in the `types` file as well

File: packages/solid/components/Column/Column.tsx [hunk 2]
```diff
@@ -33,8 +33,6 @@ const Column: Component<ColumnProps> = (props: ColumnProps) => {
   return (
     <ScrollableView
       {...props}
-      gap={props.itemSpacing}
-      transition={props.itemTransition}
```
Comment (anthony9187) [inline packages/solid/components/Column/Column.tsx:37]: these need to be removed from the interface in the `types` file as well

File: packages/solid/.storybook/preview-head.html [hunk 3]
```diff
@@ -42,4 +42,8 @@
   .sb-main-padded {
     overflow: hidden;
   }
+
+  .sb-show-main{
+    background-color: #1b1c1d;
+  }
```
Comment (anthony9187) [inline packages/solid/.storybook/preview-head.html:48]: what's this doing?

Comment (chiefcll) [reply packages/solid/.storybook/preview-head.html:48]: Thats for the popout - sets the background so it's not white.

https://rdkcentral.github.io/solid-ui/iframe.html?args=&id=components-artwork--image&viewMode=story

**Review by anthony9187 — APPROVED** [review]

**Pull request merged**